src/event_loop.rs

```diff
@@ -185,6 +185,28 @@ impl<T> EventLoop<T> {
             event_loop_proxy: self.event_loop.create_proxy(),
         }
     }
+
+    /// Change [`DeviceEvent`] filter mode.
+    ///
+    /// Since the [`DeviceEvent`] capture can lead to high CPU usage for unfocused windows, winit
+    /// will ignore them by default on Linux/BSD. This method allows changing this filter at
```

Comment:
Wayland is not implemented yet
```suggestion
    /// will ignore them by default on X11. This method allows changing this filter at
```

Reply:
I've intentionally chosen this wording because they already get ignored on Wayland, simply because there is no other option.